nxsdk_modules_contrib/pelenet/LICENSE.md

```diff
@@ -0,0 +1,11 @@
+Copyright 2019 Carlo Michaelis
```

Comment:
Please delete this LICENSE file. We include a LICENSE https://github.com/intel-nrc-ecosystem/models/blob/master/LICENSE.

Instead please modify the README https://github.com/intel-nrc-ecosystem/models/blob/master/README.md Acknowledgement Section with your network module and authorship.